terraform/root_production.tf

```diff
@@ -37,7 +37,7 @@ module "jenkins_ec2_prod" {
   name                = "JenkinsProduction"
   subnet_id           = module.jenkins_vpc.private_subnets[1]
   security_group_id   = module.jenkins_ec2_security_group.security_group_id
-  attach_policies     = { ec2_policy = module.jenkins_ec2_policy.policy_arn, cloudwatch_agent_policy = module.jenkins_cloudwatch_agent_policy.policy_arn }
+  attach_policies     = { ec2_policy = module.jenkins_ec2_policy.policy_arn, cloudwatch_agent_policy = module.jenkins_cloudwatch_agent_policy.policy_arn, publish_policy = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:policy/TDRJenkinsPublishPolicy" }
```

Comment:
Does this policy need to be added to the Jenkins Intg instance as well?

Reply:
Ah good spot yeah, I've added it in.